docs/tokenomics/nns/proposal-requirements.md

```diff
@@ -0,0 +1,206 @@
+    - DISSOLVING: In this state, the neuron’s effective dissolve delay decreases with time. While dissolving, the neuron’s age is considered zero. Eventually it will reach the DISSOLVED state. The method `stop_dissolving` can be called to transfer the neuron to the LOCKED state, and the neuron will start aging again. The method `increase_dissolve_delay` can be used to increase the dissolve delay, but this will not stop the timer or affect the age of the neuron.
+    - DISSOLVED: In the dissolved state, the neuron’s stake can be disbursed using the disburse method. It cannot vote as its dissolve delay is considered to be zero. If the method `increase_dissolve_delay` is called in this state, the neuron will become locked with the specified dissolve delay and start aging again. Neuron holders have an incentive not to keep neurons in the dissolved state for a long time: if the holders want to make their tokens liquid, they disburse the neuron’s stake, and if they want to earn voting rewards, they increase the dissolve delay. If these incentives turn out to be insufficient, the NNS may decide to impose further restrictions on dissolved neurons.
+
+* **ControlByProposals (boolean)** (true if the neuron has a non-empty list of followers on the #NeuronManagement topic): if a neuron specifies followees on the ManageNeuron topic, it can be managed by proposals of type ManageNeuron (#NeuronManagement), which may only be voted upon by the neuron’s followers. This provides a foundation for the management of highly security-sensitive neurons since it allows them to be maintained without hot keys or the secret key of the principal, which can be kept in cold storage or even destroyed (so long as the associated balance of ICP needs never be unlocked). For example, the Internet Computer Association or other known neurons might publicize the address of special neurons that will be made to vote according to their wishes, so that others can configure their neurons to follow them and leverage their expertise and efforts in governance. One problem with such practices is that they introduce the risk that secret keys used in the management of the publicized neurons might be compromised, allowing hackers to take control and “trick” large numbers of following neurons into voting according to their wishes. If the publicized neurons have admin proposals enabled, however, then they can be administered by the neurons they follow (their followers), which are typically controlled by a large number of team members who cannot be simultaneously extorted, without any need for the usage of secret keys whatsoever.
+
+### Commands
```

Comment:
there are not all neuron commands. I think it makes sense to just mention the most important ones here, but maybe it would be good to explicitly say so? 
One could link to the github link corresponding to here https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/nns/governance/proto/ic_nns_governance/pb/v1/governance.proto?L829 where they are all defined.